enterprise/server/test/integration/ci_runner/ci_runner_test.go

```diff
@@ -230,7 +230,7 @@ func TestCIRunner_Push_WorkspaceWithCustomConfig_RunsAndUploadsResultsToBES(t *t
 	app := buildbuddy.Run(t)
 	runnerFlags = append(runnerFlags, app.BESBazelFlags()...)
 
-	result := invokeRunner(t, runnerFlags, []string{}, wsPath)
+	result := invokeRunner(t, runnerFlags, []string{"USER=root", "HOME=/root"}, wsPath)
```

Comment:
I think these should not be needed due to ensureHomeDir() and ensureUser() in `ci_runner/main.go` - what happens if these are omitted?

Reply:
I tried really hard to reproduce this from an old commit I found in my reflog... but could not.

Will revert!